packages/test/test-end-to-end-tests/src/test/batching.spec.ts

```diff
@@ -381,6 +381,23 @@ describeFullCompat("Flushing ops", (getTestObjectProvider) => {
         });
     });
 
+    describe("Batch validation when using getPendingLocalState()", () => {
+        beforeEach(async () => {
+            await setupContainers({ flushMode: FlushMode.TurnBased, enableOfflineLoad: true });
+        });
+        it("should not allow incomplete batch during orderSequentially", async () => {
+            dataObject1.context.containerRuntime.orderSequentially(() => {
+                dataObject1map1.set("key1", "value1");
+                dataObject1map2.set("key2", "value2");
+                assert.throws(() => {
+                    container1.closeAndGetPendingLocalState();
```

Comment:
A test that directly calls `dataObject1.context.containerRuntime.getPendingLocalState()` as well would be nice

Reply:
It would actually be preferred IMO

Reply:
code complains about property getPendingLocalState does not exist on IContainerRuntimeBase